apisix/utils/log-util.lua

```diff
@@ -64,7 +66,10 @@ local function get_full_log(ngx, conf)
         else
             local body_file = ngx.req.get_body_file()
             if body_file then
-                log.request.body_file = body_file
+                local file = open(body_file, "rb")
+                local content = file:read "*a"
```

Comment:
Need to check if `read` is successful like `local content, err = file:read "*a"`